README.md

```diff
@@ -59,21 +59,21 @@ If you use (or want to use) [docker](http://docker.com), you can get a fully ins
 $ docker run -ti akamaiopen/cli
 ```
 
-The container contains Akamai CLI, as well as the `purge` and `property` subcommands pre-installed.  
+The container contains Akamai CLI, as well as the `purge` and `property` subcommands pre-installed.
 
 > **Note**: When setting up your `.edgerc`, the `purge` subcommand defaults to the `ccu` credentials section, while the `property` subcommand uses the `papi` section. These can be changed using the `--section` flag.
 
 ### Compiling from Source
 
 If you want to compile it from source, you will need Go 1.7 or later, and the [Glide](https://glide.sh) package manager installed:
 
-1. Fetch the package:  
```

Comment:
The double spaces here are required for markdown to break to a newline. I assume this is an Editor trimming extraneous trailing whitespace, but in this case it's actually necessary.